mysql-test/suite/auth_sec/t/install_keyring_vault.test

```diff
@@ -0,0 +1,20 @@
+--disable_ps_protocol
```

Comment:
Yes, it is exactly what I'm proposing. Depending on the amount of changes, the merges would not be that complicated (if it's say e.g. s/keyring_file/$keyring_plugin/g or similar), and would pick up any new upstream tests. Can you please take a look how a conversion of a small test would look like?